src/solver/hydro/management/daily.cpp

```diff
@@ -149,11 +149,13 @@ struct DebugData
                              uint y,
                              const Data::AreaName& areaName) const
     {
-        std::ostringstream buffer;
-        auto path = fs::path("debug") / "solver" / std::to_string(1 + y) / "daily."
-                    / areaName.c_str() / ".txt";
+        std::ostringstream filename;
+        filename << "daily." << areaName.c_str() << ".txt";
+        auto path = fs::path("debug") / "solver" / std::to_string(1 + y) / filename.str();
```

Comment:
```suggestion
        auto path = fs::path("debug") / "solver" / std::to_string(1 + y) / "daily." / (areaName+=".txt").c_str();
```

Reply:
I don't like `areaName += ".txt"`, I prefer `areaName + ".txt"`

Reply:
I didn’t find operator+ in Yuni::CString

Reply:
What I did : 
```c++
auto path = fs::path("debug") / "solver" / std::to_string(1 + y)
                          / ("monthly." + area.name + ".txt").c_str();
```
**Yuni** can be powerful sometimes